util/type_traits.h

```diff
@@ -61,6 +56,6 @@ using is_cl_float_type =
                        std::is_same<sycl::cl_double, T>::value ||
                        std::is_same<sycl::cl_half, T>::value>;
 
-} // namespace
+}  // namespace
```

Comment:
Probably we can discuss why do we have this configuration: https://github.com/KhronosGroup/SYCL-CTS/blob/SYCL-2020/.clang-format#L41 and whether we want to change it. I'm fine with current existing trailing comment style (i.e. 1 space).

Reply:
I'm ok with changing format, but even right now not all files corresponding to the current format. In case of changing, we probably should run `clangformat.py` to format all files at once.